evap/contributor/forms.py

```diff
@@ -91,4 +93,5 @@ def save(self, *args, **kw):
 
 
 class DelegateSelectionForm(forms.Form):
-    delegate_to = UserModelChoiceField(label=_("Delegate to"), queryset=UserProfile.objects.exclude_inactive_users())
+    delegate_to = UserModelChoiceField(label=_("Delegate to"),
+                                       queryset=UserProfile.objects.exclude_inactive_users().exclude(is_proxy_user=True))
```

Comment:
exclude_proxy_users()? i think you might actually inline and remove that method (not sure why we have `exclude_inactive_users` actually, right now i don't see much of a benefit), but if you keep it, lets be consistent in using it :)

Reply:
we want to exclude inactive users from selection lists - making users inactive has more or less the only effect of not including them into lists anymore.

Reply:
sure. i was wondering why we have that exclude in a separate method, but it's fine.